mk/targets.mk

```diff
@@ -46,6 +46,10 @@ pxf: HAL_BOARD = HAL_BOARD_LINUX
 pxf: TOOLCHAIN = BBONE
 pxf: all
 
+bebop: HAL_BOARD = HAL_BOARD_LINUX
+bebop: TOOLCHAIN = BBONE
```

Comment:
is there any reason to add the parrot toolchain if you are using the BBONE one? Wouldn't be better to delay the addition when it's actually going to be used?